integration-tests/ledger-signer.spec.ts

```diff
@@ -92,7 +92,7 @@ CONFIGS().forEach(({ lib, setup, rpc }) => {
         done();
       });
 
-      it('Should get the correct public key and public key hash of the Ledger for tz3 curve and path having 1 as account value', async (done) => {
+      it('As I User I want to verify that that Ledger will provide correct public key and public key hash for tz3 curve and path having 1 as account value', async (done) => {
```

Comment:
Should we use the same format as the other files "Verify that..." without "As a User I want to"